### PR TITLE
Update async workflow queue idl change

### PR DESCRIPTION
**Pull request by Shaddoll**
**What changed?**
Update async workflow queue idl change

**Why?**

**How did you test it?**
unit tests

**Potential risks**

**Release notes**

**Documentation Changes**

### CODE REVIEW

File: common/asyncworkflow/queue/provider/provider.go [hunk 1]
```diff
@@ -38,6 +38,11 @@ type (
 		MetricsClient metrics.Client
 	}
 
+	QueueID struct {
```
Comment (taylanisikdemir) [inline common/asyncworkflow/queue/provider/provider.go:41]: Can you add comments on this struct and fields to explain what they are?

File: common/asyncworkflow/queue/provider/provider.go [hunk 2]
```diff
@@ -38,6 +38,11 @@ type (
...
+		Type string
+		ID   interface{}
```
Comment (taylanisikdemir) [inline common/asyncworkflow/queue/provider/provider.go:43]: why `interface{}`?

**Review by taylanisikdemir — APPROVED** [review]
As long as tests pass the changes look good to me. The plumbing of new fields is not trivial so thanks for handling this.

File: common/asyncworkflow/queue/provider/provider.go (hunk 1 above)
Comment (Shaddoll) [reply common/asyncworkflow/queue/provider/provider.go:41]: I accidentally committed this change. I'll remove this

**Pull request merged**